sysl2/sysl/integration/viewGen.go

```diff
@@ -0,0 +1,593 @@
+package integration
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/anz-bank/sysl/src/proto"
+	"github.com/anz-bank/sysl/sysl2/sysl/utils"
+)
+
+type IntsParam struct {
+	apps         []string
+	highlights   map[string]struct{}
+	integrations []*AppDependency
+	app          *sysl.Application
+	endpt        *sysl.Endpoint
+}
+
+type Args struct {
+	title     string
+	project   string
+	clustered bool
+	epa       bool
+}
+
+type IntsDiagramVisitor struct {
+	m          *sysl.Module
+	sb         *strings.Builder
+	highlights map[string]struct{}
+	symbols    map[string]*_var
+	topSymbols map[string]*_topVar
+	project    string
+}
+
+type _var struct {
+	label string
+	alias string
+}
+
+type _topVar struct {
+	topLabel string
+	topAlias string
+}
+
+type Statements struct {
+	stmtSlice []*sysl.Statement
```

Comment:
@JackLeeHal Why do you need to introduce another struct `Statements`?

Reply:
Need to find attributes of callStatement from endpoint witch start the call. So I build a statement slice recursively and traverse it to collect attributes.